### DISCUSSION TITLE
Enable `solid` just for sub-package like packages/app

### DISCUSSION THREAD

**Discussion by lockinmarv** [antfu/eslint-config#625] [Q&A]
Hey! I can't figure out (or I don't get it) how I can enable `solid` rules just for a specific package while having `react` enabled for the rest. Is this possible?

This is my current eslint.config.js:
```
import antfu from '@antfu/eslint-config'

export default antfu(
  {
    formatters: true,
    astro: true,
    react: true
  }
)
```

`solid` rules should be enabled for every `.tsx` file inside `packages/app`, the rest should be handled by the React rules.

**Comment by Dimava** [comment]
@m4rvr see https://github.com/antfu/eslint-config/blob/main/src/configs/solid.ts
It has `files` option where you can put globs for your Solid files
Same with React where you can put !globs for your Solid files

**Reply by lockinmarv** [reply]
Thank you, this works 🤝

**Reply by lockinmarv** [reply]
It probably works when stitching everything together manually via the functions like `solid()` or `react()`, but it doesn't work when using the presets. The types are also just defined as `boolean | OptionsOverrides`.

Is there any reason the `files` option isn't supported in there? @Dimava

**Reply by Dimava** [reply]
🤔 
I guess you should make promote this to an issue @m4rvr 
Not being able to use different config sets on different packages may be a general problem that could use some discussion

For now try
```ts
antfu({
 ....
 react: false,
 solid: false,
}, ...react(...), ...solid(...))
```

**Reply by lockinmarv** [reply]
Yeah, indeed. Will do, thanks!

Just a note, this didn't seem to work (the spread is the issue)
```ts
antfu({
 ....
 react: false,
 solid: false,
}, ...react(...), ...solid(...))
```

Just doing this works though:
```ts
antfu({
 ....
 react: false,
 solid: false,
}, react(...), solid(...))
```
